numba_cuda/numba/cuda/tests/cudadrv/test_reset_device.py

```diff
@@ -4,11 +4,11 @@
 import threading
 from numba import cuda
 from numba.cuda.cudadrv.driver import driver
-from numba.cuda.testing import unittest, ContextResettingTestCase
+from numba.cuda.testing import unittest, CUDATestCase
 from queue import Queue
 
 
-class TestResetDevice(ContextResettingTestCase):
+class TestResetDevice(CUDATestCase):
```

Comment:
The comment (and code) below suggests that the context on the main thread is unaffected (it creates new threads for the tests) so I think there should be no need to reset the context again after the test has run.